prototype_source/pt2e_quant_qat.rst

```diff
@@ -0,0 +1,476 @@
+(prototype) PyTorch 2.0 Export Quantization-Aware Training (QAT)
```

Comment:
please don't forget to update 2.0 to 2 in this file

Reply:
oh whoops